### PR TITLE
Added Darwinia network

### PR DESCRIPTION
**Pull request by isunaslabs**
Added Darwinia network

### CODE REVIEW

File: packages/app-config/src/chains/darwinia.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const darwinia: ChainConfig = {
   name: "Darwinia",
   displayName: "Darwinia",
   explorerURLs: ["https://darwinia.subscan.io/"],
-  httpsURLs: ["https://rpc.darwinia.network"],
+  httpsURLs: ["https://cors.zimjs.com/http://g1.dev.darwinia.network:10000"],
```
Comment (hackfisher) [inline packages/app-config/src/chains/darwinia.ts]: @isunaslabs Fix this.

File: packages/app-config/src/chains/darwinia.ts [hunk 2]
```diff
@@ -27,7 +27,20 @@ export const darwinia: ChainConfig = {
   chainId: 46,
   prefix: 18,
   substrate: {
-    graphQlURL: "https://subql.darwinia.network/subql-apps-darwinia/",
-    wssURL: "wss://rpc.darwinia.network",
+    graphQlURL: "https://api.subquery.network/sq/isunaslabs/darwinia2",
```
Comment (hackfisher) [inline packages/app-config/src/chains/darwinia.ts]: @fewensa review this.

**Review by hackfisher — CHANGES_REQUESTED** [review]

File: packages/app-config/src/chains/darwinia.ts (hunk 1 above)
Comment (isunaslabs) [reply packages/app-config/src/chains/darwinia.ts]: fixed

**Review by hackfisher — APPROVED** [review]
@hujw77

**Pull request merged**